configs/13.0/packages/gcc/stage_optimized

```diff
@@ -36,6 +36,20 @@ atcfg_pre_configure() {
 		  ${utilities}/get_glibc_version.sh)
 }
 
+# Required pre build hacks
+atcfg_pre_hacks() {
+    # The machine_name fix done by GCC's fixincludes step can break
+    # some system headers. As noted in the source file
+    #   ${GCC_SRC}/fixincludes/fixinc.in line 135
+    # the build should work properly without it, so we can skip it
+    # in order to avoid further surprises.
+    case_powerpc='\
+    powerpc*-*-linux-*)\
+        test -f ${MACRO_LIST} &&  echo > ${MACRO_LIST}\
+        ;;'
+    sed -i -e "/\*-\*-vxworks\*)/ i ${case_powerpc}" \
+        "${ATSRC_PACKAGE_WORK}/fixincludes/fixinc.in"
+}
```

Comment:
Isn't it easier to apply a patch to GCC?

Reply:
@tuliom True. I did it this way as there is already another sed-based hack there. 

How should I proceed with the patch? Just add the `.patch` to `[...]/packages/gcc` dir and apply it inside `gcc/sources:atsrc_apply_patches()`?

Reply:
@mscastanho No.  We use another repository for that: https://github.com/powertechpreview/powertechpreview

Here is an example on how we do that:
https://github.com/powertechpreview/powertechpreview/commit/9b646544919d52be95d0190d13187d6ce872ae4d

And in the Advance Toolchain code:
https://github.com/advancetoolchain/advance-toolchain/commit/18fe1b4c5d4072e5801d5d4a7a9bfc1de9874acf